test/farm/setup_suite.bash

```diff
@@ -0,0 +1,14 @@
+# -*- bash -*-
+
+load helpers.bash
+
+function setup_suite(){
+    # only set up the podman farm before the first test
+    run podman system connection add --identity /home/$ROOTLESS_USER/.ssh/id_rsa test-node $ROOTLESS_USER@localhost
```

Comment:
This and all lines below should be `run_podman`, with underscore, not space.

Also, this squicks me out SO MUCH. @containers/podman-maintainers has anyone considered adding a `--podman-path` option to `system connection add`?

Reply:
Spin up a 1minutetip, then on my laptop:
```console
$ bin/podman system connection add --identity /.... foo root@A.B.C.D
Error: bash: line 1: podman: command not found
: Process exited with status 127
```

Reply:
Fixed